src/main.ts

```diff
@@ -36,6 +38,8 @@ fetchConfig("config.json").then((config: any) => {
     throw new Error("Please specify lokapiHost in 'config.json'")
   }
 
+  appName = config.appName
```

Comment:
@seegwen i understand that you catch the appName from ```config.json``` but it looks that you don't do anything with it ?

Reply:
Forgot to include smth in my PR, thanks for being vigilant !